nsv13/code/modules/overmap/types/syndicate.dm

```diff
@@ -429,6 +429,44 @@
 
 /obj/structure/overmap/syndicate/ai/kadesh/on_interdict()
 	add_sensor_profile_penalty(150, 10 SECONDS)
+	
+/obj/structure/overmap/syndicate/ai/fistofsol
+	name = "\proper SSV Fist of Sol"
+	icon = 'nsv13/icons/overmap/syndicate/tuningfork.dmi'
+	icon_state = "tuningfork"
+	desc = "A terrifying vessel packing every inch of the Syndicate's abhorrent arsenal."
+	mass = MASS_LARGE
+	sprite_size = 128
+	bound_width = 128
+	bound_height = 128
+	damage_states = FALSE
+	max_weapon_range = 85
+	obj_integrity = 5000
+	max_integrity = 5000
+	integrity_failure = 5000
+	speed_limit = 16
+	flak_battery_amount = 3
+	max_tracking_range = 90
+	armor = list("overmap_light" = 99, "overmap_medium" = 65, "overmap_heavy" = 40)
+	ai_controlled = TRUE
+	shots_left = 500
+	missiles = 30
+	ai_behavior = AI_AGRESSIVE
+	ai_flags = AI_FLAG_ELITE | AI_FLAG_BATTLESHIP | AI_FLAG_SWARMER
```

Comment:
swarmer probably isn't the greatest ai flag for them to have as it overrides behavior and is primarily meant for light vessels. Destroyer and Battleship synergize, on the other hand.